website/docs/components/form/file-input/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,15 @@
+## Conformance rating
+
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
+
+## Applicable WCAG Success Criteria
+
+This section is for reference only. This component intends to conform to the following WCAG Success Criteria:
+
+<Doc::WcagList @criteriaList={{array "..." }} />
```

Comment:
This list is missing.

Reply:
Sorry about that. I added a list. Could you please let me know if I need to add or remove anything to it?

Reply:
@MelSumner Could you please double check the list I added to let me know if it's ok? Thanks!